### PR TITLE
WD: Bookmarks

### PR DESCRIPTION
**Pull request by Prabhav-Sharma**
## Bookmarks
I've added a bookmarks page, which allows the user to view all their bookmarks in one place and perform all post operations as available otherwise.

### Changes
1. Fix font text size for loaders and page heading on post and edit page for smaller screen sizes
2. Fix styles for action buttons on post and comment card components
3. Fix bug in bookmark button that was leading to multiple unnecessary re-renders.
4. Add active/inactive styling for NavLinks in SideNav component
5. Add bookmarks page

Preview for bookmarks page: https://deploy-preview-4--trippies.netlify.app/bookmarks

### CODE REVIEW

File: src/router/Router.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ function Router() {
       <Route path="/mockman" element={<MockAPI />} />
       <Route path="/login" element={<Login />} />
```
Comment (sharathnair9999) [inline src/router/Router.js]: `/` is not explicitly needed while defining routes for v6. You can omit those

**Review by sharathnair9999 — APPROVED** [review]
Good work man

**Review by shivanigangadharan — APPROVED** [review]
Changes in code look good, can merge. Just not able to see the correct link of bookmarks page through the deploy preview. Maybe you can add it in a comment.

File: src/router/Router.js (hunk 1 above)
Comment (Prabhav-Sharma) [reply src/router/Router.js]: Thank you, I didn't know this, I've removed them now ^_^

**Pull request merged**